channels/rdpecam/client/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP cmake build script
+#
+# Copyright 2024 Oleg Turovski <oleg2104@hotmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+define_channel_client("rdpecam")
+
+set(${MODULE_PREFIX}_SRCS
+	camera_device_enum_main.c
+	camera_device_main.c
+	encoding.c
+)
+
+set(${MODULE_PREFIX}_LIBS
+	freerdp winpr
+)
+
+if(WITH_MS_RDPECAM_CLIENT)
```

Comment:
where is this `option` defined?

Reply:
`WITH_MS_RDPECAM_CLIENT` needs to be specified explicitly when cmake is invoked, such as:
`-DWITH_MS_RDPECAM_CLIENT=ON`

Reply:
use https://cmake.org/cmake/help/v3.12/command/option.html to make it known (easier to discover if it is not some hidden magic)

Reply:
Done